atomic_reactor/plugins/exit_store_metadata_in_osv3.py

```diff
@@ -204,6 +205,11 @@ def run(self):
             else:
                 self.log.error("Unknown result from add_help plugin: %s", help_result)
 
+        pulp_pull_results = self.workflow.postbuild_results.get(PulpPullPlugin.key)
+        if pulp_pull_results:
+            build_id, media_types = pulp_pull_results
```

Comment:
This will change the value of `build_id` that was set at the beginning of this method? If this is not intentional, please use something like this: `_, media_types = pulp_pull_results`.
Otherwise, I think a comment is needed explaining why we want to reset the `build_id` in this case.

Reading pulp_pull, this appears to actually be the `image_id`.